<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObserverDialog</class>
 <widget class="QDialog" name="ObserverDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Reset">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="ProgressLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Value:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Cart" name="cart">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <row/>
   <column/>
   <column/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Cart</class>
   <extends>QTableWidget</extends>
   <header location="global">cart.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
